src/Gui/WorkbenchSelector.h

```diff
@@ -128,6 +130,14 @@ class GuiExport WorkbenchTabWidget : public QWidget
             setProperty("style", QString::fromUtf8(workbenchItemStyleToString(itemStyle)));
         }
 
+        QSize iconSize() const {
+	    auto size = WindowParameter::getDefaultParameter()->GetGroup("General")->GetInt("ToolbarIconSize", 16);
+	    if (size > 16) {
```

Comment:
Can we lose the magic numbers here?

Reply:
Out of curiosity, can't the icon itself be scaled by the theme itself via css here? (I have no experience with freecad theming, but it looks odd that this needs to be handled via code when we know some themes might need different values)

Reply:
Not in a way that it is aware of preferences, I think that it is possible to override it with constant size but themes in general lack a ton of abilities and some things must be defined in code. This is sad truth - we hope that we will be able to move to QML based solutions and come up with simpler and more consistent mechanism for themes

Reply:
> I don't see any usable default constant to use here as fallback. It seems to be set as hardcoded value in other places too.

I think @kadet1090's suggestion is a good one, let's go with that.